libstuff/libstuff.cpp

```diff
@@ -1845,11 +1845,24 @@ bool S_recvappend(int s, string& recvBuffer) {
 bool S_sendconsume(int s, string& sendBuffer) {
     SASSERT(s);
     // If empty, nothing to do
-    if (sendBuffer.empty())
+    if (sendBuffer.empty()) {
         return true; // Assume no error, still alive
+    }
+
+    if (SStartsWith(sendBuffer, "ESCALATE_RESPONSE")) {
+        SData tempData;
+        tempData.deserialize(sendBuffer);
+        string id = tempData["id"];
+        SINFO("Sending an ESCALATE_RESPONSE for id " << id);
+    }
+
+    // Timer for tracking how long the call to send is taking to debug slow ESCALATE_RESPONSEs
+    chrono::steady_clock::time_point start = chrono::steady_clock::now();
 
     // Send as much as we can
     ssize_t numSent = send(s, sendBuffer.c_str(), (int)sendBuffer.size(), MSG_NOSIGNAL);
+    SINFO("Send() took " << chrono::duration_cast<chrono::milliseconds>(chrono::steady_clock::now() - start).count()
```

Comment:
I think this will be *extremely* high frequency and I worry it could flood all the logs.  Perhaps only log for escalate responses, and maybe only if it's over some minimum threshold (eg, 1ms)?